docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -197,19 +138,34 @@ If you don't specify any custom naming rules, the following default styles are u
 The following *.editorconfig* file contains a naming convention that specifies that public properties, methods, fields, events, and delegates must be capitalized. Notice that this naming convention specifies multiple kinds of symbol to apply the rule to, using a comma to separate the values.
 
 ```ini
```

Comment:
In order to further emphasize the difference between "setting a property which defines a symbol group or naming style" and "setting the naming rule property to the title of another element", I've reordered the property settings in the final example:

* first, the symbol group definition,
* then, the naming style definition,
* and finally, the naming rule definition, which includes setting the symbol group and naming style for the naming rule.